examples/src/main/scala/example/ziohttp/AuthExampleApp.scala

```diff
@@ -40,57 +40,32 @@ object Auth {
   }
 
   object WebSockets {
-    private val wsSession = Http.fromZIO(Ref.make[Option[String]](None))
-
     def live[R <: Auth](
       interpreter: GraphQLInterpreter[R, CalibanError]
-    ): HttpApp[R, CalibanError] =
-      wsSession.flatMap { session =>
-        val auth =
-          new Auth {
-            def currentUser: IO[Unauthorized, String]    =
-              session.get.flatMap {
-                case Some(v) => ZIO.succeed(v)
-                case None    => ZIO.fail(Unauthorized)
+    ): App[R] = {
+      val webSocketHooks = WebSocketHooks.init[R, CalibanError](payload =>
+        ZIO
+          .fromOption(payload match {
+            case InputValue.ObjectValue(fields) =>
+              fields.get("Authorization").flatMap {
+                case StringValue(s) => Some(s)
+                case _              => None
               }
-            def setUser(name: Option[String]): UIO[Unit] = session.set(name)
-          }
-
-        val webSocketHooks = WebSocketHooks.init[R, CalibanError](payload =>
-          ZIO
-            .fromOption(payload match {
-              case InputValue.ObjectValue(fields) =>
-                fields.get("Authorization").flatMap {
-                  case StringValue(s) => Some(s)
-                  case _              => None
-                }
-              case _                              => None
-            })
-            .orElseFail(CalibanError.ExecutionError("Unable to decode payload"))
-            .flatMap(user => auth.setUser(Some(user)))
-        ) ++
-          WebSocketHooks.afterInit(ZIO.failCause(Cause.empty).delay(10.seconds)) ++
-          WebSocketHooks
-            .message(new StreamTransformer[Auth, Nothing] {
```

Comment:
Any reason to remove it?

Reply:
It's not needed anymore – `FiberRef`s seem to work perfectly with websockets now 👍